src/music.ts

```diff
@@ -368,17 +366,16 @@ export class Music {
    * E.g., Given the scale C, and the note E, returns M3
    */
   getIntervalBetween(note1: number, note2: number, direction?: number): number {
-    if (direction == null) direction = 1;
+    const dir = direction == null ? 1 : direction;
```

Comment:
btw, dir is not a good name. If you don't look up to this line you ask yourself ist a directory or maybe one of these: https://www.thefreedictionary.com/words-that-start-with-dir?

Reply:
airbnb rule do not modify parameters

Reply:
I'm commenting more on name here.

Reply:
I am happy to change dir if  you provide a proposal

Reply:
for simplicity sake you could rename the parameter to directionParam or something, but the const make direction. So you don't have to change so many lines after

Reply:
That was a good idea...

Reply:
Not really, this one is just a default value, which can be done more elegantly in typescript: https://www.typescripttutorial.net/typescript-tutorial/typescript-default-parameters/